src/index.ts

```diff
@@ -52,7 +52,7 @@ export class BlockSuggestor {
   private numBlocksPerCategory: number;
 
   /**
-   * New workspace
+   * Creates the inital empty workspace
```

Comment:
This `workspace` need not be empty as the user may pass a reference to one as an argument to `init` instead of an `HTMLElement`. Also, it's not a method, but writing `create` suggests that it does something. You want a comment that describes its relation with respect to the `BlockSuggestor` object.